electrum/gui/qml/components/wizard/WCConfirmExt.qml

```diff
@@ -0,0 +1,85 @@
+import QtQuick
+import QtQuick.Layouts
+import QtQuick.Controls
+import QtQuick.Controls.Material
+
+import org.electrum 1.0
+
+import "../controls"
+
+WizardComponent {
+    id: root
+    securePage: true
+
+    valid: false
+
+    property int cosigner: 0
+
+    function checkValid() {
+        valid = false
+        var input = customwordstext.text
+        if (input == '') {
+            return
+        }
+
+        if (cosigner) {
+            // multisig cosigner
+            if (input != wizard_data['multisig_cosigner_data'][cosigner.toString()]['seed_extra_words']) {
+                return
+            }
+        } else {
+            if (input != wizard_data['seed_extra_words']) {
+                return
+            }
+        }
+        valid = true
+    }
+
+    Flickable {
+        anchors.fill: parent
+        contentHeight: mainLayout.height
+        clip: true
+        interactive: height < contentHeight
+
+        GridLayout {
+            id: mainLayout
+            width: parent.width
+            columns: 2
```

Comment:
This can be simplified to a `ColumnLayout` since the TextField below is spanning over the full width anyway.